components/fragments.md

```diff
@@ -1,6 +1,6 @@
 # Fragments
 
-The `fragment` helper allows you to define named template fragments which can be targetted while rendering.
+The `fragment` helper allows you to define named template fragments which can be targeted while rendering.
```

Comment:
This one I had to look up just to be sure,

https://www.future-perfect.co.uk/grammar-tips/just-plain-wrong/is-it-targetted-or-targeted